### PR TITLE
Add shared seeds functionality

### PR DESCRIPTION
**Pull request by dajvido**
Add `sprig_shared` for shared seeds. The generator creates additional `shared` directory where we should keep all shared seeds.

Usage:
Use `sprig_shared [...]`just like `sprig [...]` to seed from `shared` directory (default `db/seeds/shared`) 

It's possible to seed from both - `env` and `shared` directories:

```
sprig [User, Post, ...]
sprig_shared [User, ...]
```

### CODE REVIEW

**Comment by dajvido** [comment]
Travis has some gem installing problem - locally all tests passed (109 examples, 0 failures).

**Comment by ltk** [comment]
Nice, I like that approach! I'm gonna to try to take a look at Travis to debug the problem you're seeing before merging. Stay tuned.

File: lib/sprig/helpers.rb [hunk 1]
```diff
@@ -5,9 +5,12 @@ def seed_directory
...
 
     def sprig(directive_definitions)
-      hopper = []
-      DirectiveList.new(directive_definitions).add_seeds_to_hopper(hopper)
-      Planter.new(hopper).sprig
+      plant_records(directive_definitions)
+    end
+
+    def sprig_shared(directive_definitions)
+      Sprig.configuration.seeds_directory = 'shared'
```
Comment (Fosome) [inline lib/sprig/helpers.rb]: This feels like a hack - I generally think of `Sprig.configuration` options as external configuration.

In this instance it looks it is being used as global state/variables.  Fair?

Comment (dajvido) [inline lib/sprig/helpers.rb]: Yeah, I was not sure about this specific line, but I found it as clearest way to do it without modifying the whole architecture.

Any ideas to deal with the helper?
```
    def seed_directory
      Sprig.configuration.directory
    end
```

**Comment by dajvido** [comment]
@Fosome @ltk I hope you enjoy the changes

**Comment by dajvido** [comment]
@Fosome @ltk hi guys, are you alive?

**Comment by ltk** [comment]
Sorry @dajvido! I promise we'll get to this before the end of the week.

**Comment by dajvido** [comment]
@ltk sorry for another mention, but which week? 😄

**Comment by efatsi** [comment]
Merged. @dajvido a crisp high five to you for keeping this alive while simultaneously making fun of @ltk.

**Comment by dajvido** [comment]
Yay 🎉 ! That are good news, thank you @efatsi and @ltk 🙂

**Pull request merged**